lightning/src/chain/keysinterface.rs

```diff
@@ -342,6 +340,20 @@ pub trait KeysInterface: Send + Sync {
 	fn get_channel_id(&self) -> [u8; 32];
 }
 
+#[derive(Clone)]
+/// Holds late-bound channel data.
+/// This data is available after the channel is known to be accepted, either
+/// when receiving an open_channel for an inbound channel or when
+/// receiving accept_channel for an outbound channel.
+pub struct AcceptedChannelData {
+	/// Remote public keys and base points
+	pub(crate) remote_channel_pubkeys: ChannelPublicKeys,
+	/// Remote to_self_delay
```

Comment:
Irrespective of name changes, can you expand the comments to describe what the value really means - eg "The to_self_delay value specified by our counterparty and applied on locally-broadcastable transactions, ie the amount of time that we have to wait to recover our funds if we broadcast a transaction. You'll likely want to pass this to the ln::chan_utils::build*_transaction functions when signing local transactions."

Reply:
done